.prettierrc

```diff
@@ -1,5 +1,9 @@
 {
   "semi": false,
   "arrowParens": "always",
-  "singleQuote": true
-}
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "endOfLine": "lf",
+  "trailingComma": "es5"
+}
```

Comment:
Googled for ne line at the end of the file and found: In VS Code you can set "files.insertFinalNewline": true and Prettier won't override this setting.